docs/getting-started.md

```diff
@@ -32,6 +32,14 @@ out directly.
 
 Releases are also available from the `zrok` project repository on GitHub at https://github.com/openziti/zrok/releases/latest
 
+You need the right download for your OS and CPU. This can be confusing for Raspberry Pi users because their Raspberry Pi OS could be 32bit or 64bit. Linux users can find the right version by looking up the result of `uname -m` in this table.
+
+|                            |macOS       |Windows      |Linux       |
+|----------------------------|------------|-------------|------------|
+| AMD (x86_64) 64bit         |darwin-amd64|windows-amd64|linux-amd64 |
+| ARM (aarch64, arm/v8) 64bit|darwin-arm64|             |linux-arm64 |
```

Comment:
a.) windows and mac shouldn't even be in this table
b.) putting the values in parentheticals made me skip over them entirely
c.) this information should be on the download page
d.) it doesn't help me since line 33 above says "go to github"...

i don't think it really fits/helps out